.env.docker

```diff
@@ -0,0 +1,37 @@
+APP_ENV=docker
+APP_KEY=base64:cnJls/mXUDquZgRhN+HHVDlaGlwCL/uStpuasXHlv/8=
+APP_DEBUG=false
+APP_LOG_LEVEL=critical
+APP_URL=http://127.0.0.1:80
+
+DB_CONNECTION=mysql
+DB_HOST=127.0.0.1
+DB_PORT=33061
+DB_DATABASE=homestead
+DB_USERNAME=root
+DB_PASSWORD=secret
+
+BROADCAST_DRIVER=log
+CACHE_DRIVER=file
+SESSION_DRIVER=file
+QUEUE_DRIVER=sync
+
+REDIS_HOST=127.0.0.1
+REDIS_PASSWORD=null
+REDIS_PORT=6379
+
+MAIL_DRIVER=smtp
+MAIL_HOST=mailtrap.io
+MAIL_PORT=2525
+MAIL_USERNAME=null
+MAIL_PASSWORD=null
+MAIL_ENCRYPTION=null
+
+PUSHER_APP_ID=
+PUSHER_KEY=
+PUSHER_SECRET=
+
+# Numbers are in years
+OLD_USER_SAVE_PERIODE=
+APPLICATION_RESPONSE_SAVE_PERIODE=
+
```

Comment:
Zie env.example in de root folder, hier missen nog wat dingen

Reply:
Ik zal ze toevoegen